electronics_lib/DcDcConverters.py

```diff
@@ -1,5 +1,81 @@
 from electronics_abstract_parts import *
 
+class Tps61023_Device(DiscreteChip, FootprintBlock):
+  @init_in_parent
+  def __init__(self, current_draw: RangeLike = RangeExpr()):
+    super().__init__()
+    self.vin = self.Port(VoltageSink(
+      voltage_limits=(0.5, 5.5)*Volt,
+      current_draw=current_draw
+    ))
+    self.gnd = self.Port(Ground())
+    self.sw = self.Port(VoltageSource())
+    self.fb = self.Port(AnalogSink(impedance=(8000, float('inf')) * kOhm))
+    self.vout = self.Port(Passive()) # TODO model as voltage source
+
+  def contents(self) -> None:
+    super().contents()
+    self.footprint(
+      'U', 'Package_TO_SOT_SMD:SOT-23-6',
+      {
+        '1': self.fb,
+        '2': self.vin, # en
+        '3': self.vin,
+        '4': self.gnd,
+        '5': self.sw,
+        '6': self.vout,
+      },
+      mfr='Texas Instruments', part='TPS61023',
+      datasheet='https://www.ti.com/lit/ds/symlink/tps61023.pdf'
+    )
+
+class Tps61023(DiscreteBoostConverter, GeneratorBlock):
+
+  def contents(self):
+    super().contents()
+    self.assign(self.frequency, (0.5, 1)*MHertz)
+    self.assign(self.efficiency, (0.7, 0.97))
+
+    self.fb = self.Block(FeedbackVoltageDivider(
+      output_voltage=(580, 610) * mVolt,
+      impedance=(1, 300) * kOhm,
```

Comment:
I recommend setting the lower bound at 10k (unless the datasheet says otherwise?)
The main drawback of a lower impedance is higher power draw through the resistive divider. It will result in a 'stronger' signal, but past some point it doesn't matter and would just consume more power.